blazor/datagrid/filtering.md

```diff
@@ -461,15 +461,63 @@ Operator |Description |Supported Types
 StartsWith |Checks whether the value begins with the specified value. |String
 EndsWith |Checks whether the value ends with the specified value. |String
 Contains |Checks whether the value contains the specified value. |String
+DoesNotStartWith |Checks whether the value does not begin with the specified value. |String
+DoesNotEndWith |Checks whether the value does not end with the specified value. |String
+DoesNotContain |Checks whether the value does not contain the specified value. |String
 Equal |Checks whether the value is equal to the specified value. |String &#124; Number &#124; Boolean &#124; Date
 NotEqual |Checks for values not equal to the specified value. |String &#124; Number &#124; Boolean &#124; Date
 GreaterThan |Checks whether the value is greater than the specified value. |Number &#124; Date
 GreaterThanOrEqual |Checks whether a value is greater than or equal to the specified value. |Number &#124; Date
 LessThan |Checks whether the value is less than the specified value. |Number &#124; Date
 LessThanOrEqual |Checks whether the value is less than or equal to the specified value. |Number &#124; Date
+IsNull |Returns the values that are null. |String &#124; Number &#124; Date
+IsNotNull |Returns the values that are not null. |String &#124; Number &#124; Date
+IsEmpty |Returns the values that are empty. |String
+IsNotEmpty |Returns the values that are not empty. |String
+between |Filter the values based on the range between the start and end specified values. |Number &#124; Date
 
 > By default, the **Operator** value is **Equal** in [Columns](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.GridFilterSettings.html#Syncfusion_Blazor_Grids_GridFilterSettings_Columns) property of **GridFilterSettings** component.
 
+## Wildcard and LIKE operator filter
+
+**Wildcard** and **LIKE** filter operators filter value based on the given string pattern, and they apply to string-type columns. But it will work slightly differently.
```

Comment:
Why we use LIKE operator in all caps instead of Like

Reply:
Change as per guidelines.